---
layout: default
---


The Köster lab is focused on reproducible data analysis, algorithm engineering and Bayesian statistics in Bioinformatics.


# Members

* [Johannes Köster](https://johanneskoester.bitbucket.io) (group leader)
* [Bianca Stöcker](http://genomeinformatics.uni-due.de/people/bianca-stocker)
* Jan Forster



# Projects

## [Snakemake](https://snakemake.bitbucket.io)

Snakemake is a workflow engine and language. It aims to reduce the complexity of creating workflows by providing a fast and comfortable execution environment, together with a clean and modern domain specific specification language (DSL) in python style. Available on [Bioconda](https://bioconda.github.io/recipes/snakemake/README.html).

## [Bioconda](https://bioconda.github.io)

A distribution of bioinformatics software realized as a channel for the versatile package manager Conda. It is a multinational project with over 200 developers and over 2000 software packages.

## [PROSIC](https://prosic.github.io)

A caller for somatic variants in tumor normal pairs that leverages a latent variable model to improve the sensitivity and control the false discovery rate. Available on [Bioconda](https://bioconda.github.io/recipes/prosic/README.html).

## [MERFISHtools](https://merfishtools.github.io)

A Bayesian model for single-cell transcript (differential) expression analysis on MERFISH data. The model allows to overcome systematic biases occurring with MERFISH and provides measures of uncertainty and control of the false discovery rate in a strictly Bayesian way. MERFISHtools is a corresponding command line client and analysis library written in Rust and Python. Available on [Bioconda](https://bioconda.github.io/recipes/merfishtools/README.html).

## [Rust-Bio](https://rust-bio.github.io)

A bioinformatics library written in the [Rust language](https://rust-lang.org). The implementation provides state of the art solutions for common tasks in bioinformatics, focusing on stability by using comprehensive unit tests and continuous integration.


## Protein Hypernetworks

This project aims to model protein networks along with dependencies between interactions like allosteric effects, steric hindrance and competition on binding domains. This allows us to improve the accuracy of protein complex prediction and to simulate the effects of perturbations like knockout or overexpression.


# Location
Genome Informatics  
Institute of Human Genetics  
University of Duisburg-Essen  
University Hospital Essen  
Virchowstr. 183  
45147 Essen
Germany
