<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <title>Johannes Köster's Homepage</title>
    <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet" type="text/css">
    <link type="text/css" rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.0.0/css/bootstrap.min.css">
    <link type="text/css" rel="stylesheet" href="style.css">
</head>

<body data-spy="scroll" data-target="#header">
    <div id="header" class="navbar-fixed-top navbar navbar-default navbar-inverse" role="navigation">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        </button>
            <a href="#" class="navbar-brand">Dr. rer. nat. Johannes Köster</a>
        </div>
        <div class="collapse navbar-collapse">
            <ul class="nav navbar-nav navbar-right">
                <li class="active"><a href="#about">about me</a></li>
                <li><a href="#research">research</a></li>
                <li><a href="#contact">contact</a></li>
                <li><a href="https://twitter.com/johanneskoester" style="color: #1da1f2; font-weight: bold">twitter</a></li>
                <li><a href="https://www.researchgate.net/profile/Johannes_Koester/" style="color: #00D0AF; font-weight: bold">researchgate</a></li>
                <li><a href="https://www.xing.com/profile/Johannes_Koester3" style="color: #006567; font-weight: bold">xing</a></li>
                <li><a href="https://keybase.io/johanneskoester" style="color: #ff6f21; font-weight: bold">keybase</a></li>
                <li><a href="https://orcid.org/0000-0001-9818-9320" style="color: #a6ce39; font-weight: bold">ORCiD</a></li>
            </ul>
        </div>
    </div>

    <div class="container">
        <section id="about">
            <img src="me.png" id="imgme" class="img-rounded">
            <div>
                <p>
                    Johannes Köster is professor for <a href="https://koesterlab.github.io/">Bioinformatics and Computational Oncology (Bioinformatische Algorithmen in der Onkologie)</a> at the Institute for AI in Medicine (IKIM), University Medicine Essen, University of Duisburg-Essen. He has a focus on algorithm engineering and data analysis. Johannes studied computer science at the University of Dortmund (diploma thesis 2010, with Max Planck Institute of Molecular Physiology
                    Dortmund). Then, he did his PhD in the group of Prof. Sven Rahmann (TU Dortmund, 2015). Afterwards, Johannes was a postdoc in the groups of Prof. Shirley Liu and Prof. Myles Brown at Dana Farber Cancer Institute and Harvard University
                    (2015-2016). In 2016, Johannes moved to the lab of Prof. Alexander Schönhuth for a brief second postdoc at Centrum Wiskunde & Informatica Amsterdam, Netherlands (CWI), where he quickly received a VENI grant for an independent position.
                    In 2017, Johannes became the head of the group "Algorithms for reproducible bioinformatics" at the University of Duisburg-Essen in Germany.
                    Johannes was appointed full professor at the University of Duisbug-Essen in 2023.
                </p>
                <p>
                    Johannes' research is focused on reproducibility in three ways. First, he is the author of the popular workflow management system <a href="https://snakemake.readthedocs.io">Snakemake</a> and the founder of the <a href="https://doi.org/10.1038/s41592-018-0046-7">Bioconda</a>                    project for sustainably distributing bioinformatics software as easily installable packages. Together, these projects form the base for a large fraction of currently performed scalable and reproducible data analysis in bioinformatics.<br/>                    Second, Johannes is the author of the <a href="https://doi.org/10.1093/bioinformatics/btv573">Rust-Bio library</a>, enabling the use of the <a href="https://rust-lang.org">Rust programming language</a> for bioinformatics by providing
                    standard bioinformatics algorithms and data structures. Using Rust promotes reproducibilty by guaranteeing thread and memory safety at compile-time.<br/> Third, Johannes is working in the field of Bayesian statistics (e.g., for
                    <a href="https://doi.org/10.1186/s13059-020-01993-6">variant calling</a> and <a href="https://doi.org/10.1093/bioinformatics/bty718">single cell transcriptomics</a>) in order to provide algorithms for analysis of high-throughput data
                    while capturing and quantifying all known sources of uncertainty, thereby providing more reproducible predictions.
                </p>

                <table class="table">
                    <thead>
                        <tr>
                            <th></th>
                            <th>Position</th>
                            <th>Group</th>
                        </tr>
                    </thead>
                    <tbody>
						<tr>
                            <th>Since March 2023</th>
                            <td>Full professor</th>
                                <td><a href="https://koesterlab.github.io/">Bioinformatics and computational oncology (Bioinformatische Algorithmen in der Onkologie), Institute for AI in Medicine (IKIM), University Medicine Essen, University of Duisburg-Essen</a></td>
                        </tr>
                        <tr>
                            <th>August 2017 -</br>March 2023</th>
                            <td>Group leader</th>
                                <td><a href="https://koesterlab.github.io/">Algorithms for reproducible bioinformatics, Institute of Human Genetics, University of Duisburg-Essen</a></td>
                        </tr>
                        <tr>
                            <th>August 2016 -</br>July 2017</th>
                            <td>Researcher</th>
                                <td><a href="http://homepages.cwi.nl/~as/">Life Sciences, CWI Amsterdam</a></td>
                        </tr>
                        <tr>
                            <th>Since May 2016</th>
                            <td>Consultant</td>
                            <td><a href="https://research4.dfci.harvard.edu/brownlab/">Myles Brown, Division of Molecular and Cellular Oncology, Department of Medical Oncology, Dana-Farber Cancer Institute, Harvard Medical School</a></td>
                        </tr>
                        <tr>
                            <th>May 2016 -<br/>July 2016</th>
                            <td>Postdoc</th>
                                <td><a href="http://homepages.cwi.nl/~as/">Alexander Schönhuth, Life Sciences, CWI Amsterdam</a></td>
                        </tr>
                        <tr>
                            <th rowspan=2>April 2015 -<br/>April 2016</th>
                            <td>Postdoc</td>
                            <td><a href="https://liulab.dfci.harvard.edu/">Shirley Liu, Department of Biostatistics and Computational Biology, Dana-Farber Cancer Institute, Harvard School of Public Health</a></th>
                        </tr>
                        <tr>
                            <td>Postdoc</td>
                            <td><a href="https://research4.dfci.harvard.edu/brownlab/">Myles Brown, Division of Molecular and Cellular Oncology, Department of Medical Oncology, Dana-Farber Cancer Institute, Harvard Medical School</a></th>
                        </tr>
                        <tr>
                            <th rowspan=2>January 2011 -<br/>March 2015</th>
                            <td>PhD student</td>
                            <td><a href="http://www.rahmannlab.de">Sven Rahmann, Genome Informatics, University Duisburg-Essen</a></td>
                        </tr>
                        <tr>
                            <td>Guest member</td>
                            <td><a href="http://www.cell-adhesion.com">Eli Zamir, Systems Biology of Cell Matrix Adhesion, Max-Planck-Institute of Molecular Physiology Dortmund</a></td>
                        </tr>
                    </tbody>
                </table>

                See <a href="resources/cv.pdf">here</a> for a full CV.
            </div>
        </section>


        <section id="research">
            <h3>Research</h3>

            <section>
                <h4>Projects</h4>
                <table class="table">
                    <tbody>
                        <tr>
                            <th>Snakemake</th>
                            <td>
                                <p>Snakemake is a workflow engine and language. It aims to reduce the complexity of creating workflows by providing a fast and comfortable execution environment, together with a clean and modern domain specific specification
                                    language (DSL) in python style.</p>
                                <p class="text-right"><a href="https://snakemake.readthedocs.io">visit homepage</a></p>
                            </td>
                        </tr>
                        <tr>
                            <th>Bioconda</th>
                            <td>
                                <p>A distribution of bioinformatics software realized as a channel for the versatile package manager <a href="https://conda.pydata.org/docs">Conda</a>.</p>
                                <p class="text-right"><a href="https://bioconda.github.io">visit homepage</a></p>
                            </td>
                        </tr>
                        <tr>
                            <th>Varlociraptor</th>
                            <td>
                                <p>Flexible, uncertainty-aware variant calling with parameter free filtration via FDR control. The project aims to provide a unified theory of variant calling, across all size ranges (small, structural) and calling scenarios
                                    (germline, somatic, arbitrary).</p>
                                <p class="text-right"><a href="https://varlociraptor.github.io">visit homepage</a></p>
                            </td>
                        </tr>
                        <tr>
                            <th>MERFISHtools</th>
                            <td>
                                <p>A Bayesian model for single-cell transcript (differential) expression analysis on <a href="https://science.sciencemag.org/content/early/2015/04/08/science.aaa6090.abstract">MERFISH</a> data. The model allows to overcome
                                    systematic biases occurring with MERFISH and provides measures of uncertainty and control of the false discovery rate in a strictly Bayesian way. MERFISHtools is a corresponding command line client and analysis library
                                    written in <a href="https://rust-lang.org">Rust</a> and <a href="https://python.org">Python</a>. MERFISHtools is also available via <a href="https://bioconda.github.io">Bioconda</a>.</p>
                                <p class="text-right"><a href="https://merfishtools.github.io">visit homepage</a></p>
                            </td>
                        </tr>
                        <tr>
                            <th>Rust-Bio</th>
                            <td>
                                <p>A bioinformatics library written in the <a href="https://rust-lang.org">Rust language</a>. The implementation provides state of the art solutions for common tasks in bioinformatics, focusing on stability by using comprehensive
                                    unit tests and continuous integration.</p>
                                <p class="text-right"><a href="https://rust-bio.github.io">visit homepage</a></p>
                            </td>
                        </tr>
                        <tr>
                            <th>ALPACA</th>
                            <td>
                                <p>ALPACA is a variant caller for next-generation sequencing data that incorporates sample based filtering into the calling. This allows intuitive control of the false discovery rate with generic sample filtering scenarios.
                                    Further, it uses preprocessing and merging of BCF files to solve the N+1 problem: an existing study can be extended with new samples without redundant computations. After the preprocessing, the actual calling is a matter
                                    of seconds.</p>
                                <p class="text-right"><a href="https://github.com/johanneskoester/alpaca">visit homepage</a></p>
                            </td>
                        </tr>
                        <tr>
                            <th>PEANUT</th>
                            <td>
                                <p>PEANUT is a read mapper for DNA or RNA sequence reads. By exploiting the massive parallelism of modern graphics processors and a novel index datastructure, PEANUT achieves superior speed compared to current state of the
                                    art read mappers like BWA MEM, Bowtie2 and RazerS3, while maintaining their accuracy. It thereby allows to report both only the best hits or all hits of a read. In case of reporting all hits, PEANUT is four to ten times
                                    faster than competitors.</p>
                                <p class="text-right"><a href="https://peanut.readthedocs.org">visit homepage</a></p>
                            </td>
                        </tr>
                        <tr>
                            <th>LibModalLogic</th>
                            <td>
                                <p>LibModalLogic is a JAVA implementation of Modal Logic K and Propositional Logic. Logic formulas can be build in memory, saved to and read from MathML and formatted human readable. Reasoning is implemented by the (modal)
                                    logic tableau algorithm, including dynamic backtracking for maximum performance.</p>
                                <p class="text-right"><a href="https://libmodallogic.googlecode.com">visit at google code</a></p>
                            </td>
                        </tr>
                        <tr>
                            <th>TRMiner</th>
                            <td>
                                <p>TRMiner is a python tool that aims at scientific data curators. It allows to rapidly prune large collections of scientific publications to sentences relevant for a given mining goal, using a linear time matching algorithm.</p>
                                <p class="text-right"><a href="https://trminer.googlecode.com">visit at google code</a></p>
                            </td>
                        </tr>
                        <tr>
                            <th>Protein Hypernetworks</th>
                            <td>
                                <p>Protein Hypernetworks are an approach for endowing protein networks with interaction dependencies using propositional logic. This allows refined network based predictions of protein complexes, functional importance and
                                    functional similarity.</p>
                                <p class="text-right"><a href="https://proteinhypernetworks.googlecode.com"> visit at google code</a></p>
                            </td>
                        </tr>
                    </tbody>
                </table>
            </section>

            <section>
                <h4>Publications</h4>
                <ul>
                	<li><a class="hiddenlink" href="https://doi.org/10.1126/scisignal.ade6737">Bergkamp, N. D., van Senten, J.R., Brink, H. J., Bebelman, M. P., Bor, J. v. d., Çobanoğlu, T. S., Dinkla, K., Köster, J., Klau, G., Siderius, M., Smit, M. J. <i>A virally encoded GPCR drives glioblastoma through feed-forward activation of the SK1-S1P1 signaling axis.</i> Science Signaling 16 (2023).
</a></li>
                    <li><a class="hiddenlink" href="https://www.frontiersin.org/articles/10.3389/fgene.2022.867018">Tüns, A.I., Hartmann, T., Magin, S., González, R.C., Henssen, A.G., Rahmann, S., Schramm, A., Köster, J., 2022. Detection and Validation of Circular DNA Fragments Using Nanopore Sequencing. Frontiers in Genetics 13.</a></li>
                    <li><a class="hiddenlink" href="https://doi.org/10.12688/f1000research.29032.1">Mölder, F., Jablonski, K.P., Letcher, B., Hall, M.B., Tomkins-Tinch, C.H., Sochat, V., Forster, J., Lee, S., Twardziok, S.O., Kanitz, A., Wilm, A., Holtgrewe, M., Rahmann, S., Nahnsen, S., Köster, J. <i>Sustainable data analysis with Snakemake</i>. F1000Res 10, 33 (2021).</a></li>
                    <li><a class="hiddenlink" href="https://doi.org/10.1186/s13059-020-01993-6">Köster, J., Dijkstra, L. J., Marschall, T., Schönhuth, A. <i>Varlociraptor: enhancing sensitivity and controlling false discovery rate in somatic indel discovery</i>. Genome Biology 21, 98 (2020).</a></li>
                    <li><a class="hiddenlink" href="https://link.springer.com/chapter/10.1007/978-3-030-32047-8_27">Stöcker, B., Schäfer, T., Mutzel, P., Köster, J., Kriege, N., Rahmann, S. <i>Protein Complex Similarity Based on Weisfeiler-Lehman Labeling. in Similarity Search and Applications</i> 308–322 (Springer International Publishing, 2019).</a></li>
                    <li><a class="hiddenlink" href="https://doi.org/10.1093/bioinformatics/btz443">Baaijens, J. A., Van der Roest, B., Köster, J., Stougie, L., Schönhuth, A. <i>Full-length de novo viral quasispecies assembly through variation graph construction.</i> Bioinformatics</a></li>
                    <li><a class="hiddenlink" href="https://doi.org/10.1093/bioinformatics/bty718">Johannes Köster, Myles Brown, X. Shirley Liu. <i>A Bayesian Model for Single Cell Transcript Expression Analysis on MERFISH Data.</i> Bioinformatics 2018.</a></li>
                    <li><a class="hiddenlink" href="https://doi.org/10.1038/s41592-018-0046-7">Björn Grüning, Ryan Dale, Andreas Sjödin, Brad A. Chapman, Jillian Rowe, Christopher H. Tomkins-Tinch, Renan Valieris, the Bioconda Team, and Johannes Köster. <i>Bioconda: Sustainable and Comprehensive Software Distribution for the Life Sciences.</i> Nature Methods 2018.</a></li>
                    <li><a class="hiddenlink" href="https://doi.org/10.1039/c8ib00012c">Bianca Stöcker, Johannes Köster, Eli Zamir, Sven Rahmann. <i>Modeling and simulating networks of interdependent protein interactions.</i> Integr Biol 2018.</a></li>
                    <li><a class="hiddenlink" href="https://doi.org/10.1186/s12859-018-2139-9">MacIntosh Cornwell, Mahesh Vangala, Len Taing, Zachary Herbert, Johannes Köster, Bo Li, Hanfei Sun, Taiwen Li, Jian Zhang, Xintao Qiu, Matthew Pun, Rinath Jeselsohn, Myles Brown, X. Shirley Liu and Henry W. Long. VIPER: Visualization Pipeline for RNA-seq, a Snakemake workflow for efficient and complete RNA-seq analysis. BMC Bioinformatics 2018.</a></li>
                    <li><a class="hiddenlink" href="https://link.springer.com/article/10.1007/s00441-018-2810-5">Schulte, M., Köster, J., Rahmann, S. & Schramm, A. <i>Cancer evolution, mutations, and clonal selection in relapse neuroblastoma.</i> Cell Tissue Res 2018.</a></li>
                    <li><a class="hiddenlink" href="https://www.cell.com/cancer-cell/abstract/S1535-6108(16)30090-3">Townsend, E. C., Murakami, M. A., Christodoulou, A., Christie, A. L., Köster, J., DeSouza, T. A., … Weinstock, D. M. (2016). <i>The Public Repository of Xenografts Enables Discovery and Randomized Phase II-like Trials in Mice.</i> Cancer Cell 2016</a></li>
                    <li><a class="hiddenlink" href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0843-6">Li, W., Köster, J., Xu, H., Chen, C.-H., Xiao, T., Liu, J. S., ... Liu, X. S. <i>Quality Control, Modeling, and Visualization of CRISPR Screens with MAGeCK-VISPR.</i> Genome Biology 2015.</a></li>
                    <li><a class="hiddenlink" href="https://doi.org/10.1093/bioinformatics/btv573">Köster, J. <i>Rust-Bio: a fast and safe bioinformatics library.</i> Bioinformatics 2015.</a></li>
                    <li><a class="hiddenlink" href="https://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3349.html">Schramm, A., Köster, J., Assenov, Y., Althoff, K., Peifer, M., Mahlow, E., ... Schulte, J. H. <i>Mutational dynamics between primary and relapse neuroblastomas.</i> Nature Genetics 2015.</a></li>
                    <li><a class="hiddenlink" href="https://hdl.handle.net/2003/33940">Köster, J. <i>Parallelization, Scalability, and Reproducibility in Next-Generation Sequencing Analysis.</i> PhD-Thesis, TU Dortmund, Germany 2014. ISBN: 978-3737537773.</a></li>
                    <li><a class="hiddenlink" href="https://dx.doi.org/10.7717/peerj.606">Köster, J., Rahmann, S. <i>Massively parallel read mapping on GPUs with the q-group index and PEANUT.</i> PeerJ 2014.</a></li>
                    <li><a class="hiddenlink" href="https://bioinformatics.oxfordjournals.org/content/28/19/2520.long">Köster, Johannes and Sven Rahmann. <i>Snakemake - A scalable bioinformatics workflow engine.</i> Bioinformatics 2012.</a></li>
                    <li><a class="hiddenlink" href="https://pubs.rsc.org/en/content/articlelanding/2012/ib/c2ib00126h#!divAbstract">Köster, Johannes, Eli Zamir, and Sven Rahmann. <i>Efficiently Mining Protein Interaction Dependencies from Large Text Corpora.</i> Integrative Biology 2012.</a></li>
                    <li><a class="hiddenlink" href="https://drops.dagstuhl.de/opus/volltexte/2012/3717/">Köster, Johannes and Sven Rahmann. <i>Building and Documenting Bioinformatics Workflows with Python-based Snakemake.</i> Proceedings of the GCB 2012.</a></li>
                    <li><a class="hiddenlink" href="https://onlinelibrary.wiley.com/doi/10.1002/ijc.27787/abstract">Schramm A, Köster J, Marschall T, Martin M, Heilmann M, Fielitz K, Büchel G, Barann M, Esser D, Rosenstiel P, Rahmann S, Eggert A, Schulte JH. <i>Next-generation RNA sequencing reveals differential expression of MYCN target genes and suggests the mTOR pathway as a promising therapy target in MYCN-amplified neuroblastoma.</i> Int J Cancer 2012</a></li>
                    <li><a class="hiddenlink" href="https://onlinelibrary.wiley.com/doi/10.1002/ijc.28091/abstract">Althoff K, Beckers A, Odersky A, Mestdagh P, Köster J, Bray IM, Bryan K, Vandesompele J, Speleman F, Stallings RL, Schramm A, Eggert A, Sprüssel A, Schulte JH. <i>MiR-137 functions as a tumor suppressor in neuroblastoma by downregulating KDM1A.</i> Int J Cancer. 2013</a></li>
                    <li><a class="hiddenlink" href="https://www.ncbi.nlm.nih.gov/pubmed/23098880">Sven Rahmann, Marcel Martin, Johannes H. Schulte, Johannes Köster, Tobias Marschall, Alexander Schramm. <i>Identifying transcriptional miRNA biomarkers by integrating high-throughput sequencing and real-time PCR data.</i> Methods 2012.</a></li>
                    <li><a class="hiddenlink" href="https://www.nature.com/bjc/journal/v107/n8/full/bjc2012391a.html">A Schramm, B Schowe, K Fielitz, M Heilmann, M Martin, T Marschall, J Köster, J Vandesompele, J Vermeulen, K de Preter, J Koster, R Versteeg, R Noguera, F Speleman, S Rahmann, A Eggert, K Morik and J H Schulte. <i>Exon-level expression analyses identify MYCN and NTRK1 as major determinants of alternative exon usage and robustly predict primary neuroblastoma outcome.</i> British Journal of Cancer 2012.</a></li>
                    <li><a class="hiddenlink" href="https://ls11-www.cs.uni-dortmund.de/_media/techreports/tr11-02.pdf">Köster, Johannes. <i>Propagating Interaction Logic towards Predictive Protein Hypernetworks.</i> Diploma Thesis at the Faculty of Computer Science, TU Dortmund 2010.</a>                        The thesis was honored with the <a href="http://www.cs.uni-dortmund.de/nps/de/Home/Aktuelles_nicht_angezeigt/Newsarchiv/Uhde_Preis_2011/index.html">Uhde-Award 2011</a>.</li>
                    <li><a class="hiddenlink" href="https://www.ruhr-uni-bochum.de/ifhv/documents/workingpapers/wp4_3.pdf">Engler, S., Köster, J. and Siebert, A.  <i>Farmers Food Insecurity Monitoring. Identifying situations of food insecurity and famine.</i> IFHV Working Paper 2014.</a></li>
                </ul>
            </section>

            <section>
                <h4>Talks</h4>
                <ul>
                    <li><a class="hiddenlink" href="http://slides.com/johanneskoester/aebc-2017">Keynote about workflow and tool management at AEBC 2017, Tübingen.</a></li>
                    <li><a class="hiddenlink" href="http://slides.com/johanneskoester/keynote-jk-dresden-2016">Keynote (german) about the in-silico reproducibility crisis at Helmholtz Workshop on sustainable software development, Dresden 2016.</a></li>
                    <li><a class="hiddenlink" href="https://slides.com/johanneskoester/rust-bio-mozilla">Invited talk about Rust-Bio at Rust Bay Area Meeting hosted by Mozilla, Los Angeles 2016.</a></li>
                    <li><a class="hiddenlink" href="https://slides.com/johanneskoester/prosic-hitseq-2016">Talk about genotyping somatic insertions and deletions at HiTSeq, Orlando 2016.</a></li>
                    <li><a class="hiddenlink" href="https://slides.com/johanneskoester/snakemake-bioconda-leeselab">Invited talk about Snakemake and Bioconda at Aquatic Ecosystem Research, University of Duisburg-Essen, Essen 2016.</a></li>
                    <li><a class="hiddenlink" href="https://slides.com/johanneskoester/merfishtools">Talk about MERFISHtools at the Life Science seminar, CWI Amsterdam 2016</a></li>
                    <li><a class="hiddenlink" href="https://slides.com/johanneskoester/snakemake-broad-2015">Invited talk about Snakemake at the Broad Institute, Boston 2015.</a></li>
                    <li><a class="hiddenlink" href="https://slides.com/johanneskoester/deck-1">Snakemake workshop at the German Conference on Bioinformatics, Dortmund 2015.</a></li>
                    <li><a class="hiddenlink" href="slides/jk-dresden-peanut-2014.pdf">Invited talk about PEANUT at the Workshop on Fast Data Processing on GPUs, Dresden CUDA Center of Excellence, Dresden 2014.</a></li>
                    <li><a class="hiddenlink" href="slides/jk-dtl-snakemake-2014.pdf">Invited talk about Snakemake at the DTL Focus Meeting: "NGS Production Pipelines", Dutch Techcentre for Life Sciences, Utrecht 2014.</a></li>
                    <li><a class="hiddenlink" href="slides/jk-ngs-snp-calling-2013.pdf">Guest lecture at the Faculty of Statistics, TU Dortmund, 2013 on "Detecting SNVs with next generation sequencing".</a></li>
                    <li><a class="hiddenlink" href="slides/jk-retreat-2013.pdf">Northwestern Germany Bioinformatics Retreat 2013, on novel features and algorithmic details of Snakemake.</a></li>
                    <li><a class="hiddenlink" href="slides/jk-gcb-2012.pdf">German Conference on Bioinformatics, September 2012, on "Building and Documenting Bioinformatics Workflows with Python-based Snakemake".</a></li>
                    <li><a class="hiddenlink" href="slides/jk-ismp-2012.pdf">International Symposium on Mathematical Programming 2012, on protein hypernetworks.</a></li>
                    <li><a class="hiddenlink" href="slides/jk-9th-pathways-2011.pdf">9th International Conference on Pathways, Networks and Systems Medicine on Crete 2011, on protein hypernetworks. It was honored with a Travel Award.</a></li>
                    <li><a class="hiddenlink" href="slides/jk-tcs-workshop-2011.pdf">Mini-Workshop Theoretical Computer Science in Dortmund 2011, on theoretical aspects of protein hypernetworks.</a></li>
                </ul>
            </section>
            <section>
                <h4>Poster</h4>
                <ul>
                    <li><a class="hiddenlink" href="posters/snakemake+bioconda-2016.pdf">ECCB 2016, The Hague, Netherlands. <i>Fully reproducible data analysis with Snakemake and Bioconda</i></a>
                        <li><a class="hiddenlink" href="posters/pqg-2015.pdf">PQG Conference on Single Cell Genomics, Harvard T.H. Chan School of Public Health 2015. <i>A Bayesian model for gene expression on MERFISH data</i></a>
                            <li><a class="hiddenlink" href="posters/mco-retreat-2015.pdf">Medical Oncology Retreat, Dana-Farber Cancer Institute, Harvard Medical School 2015. <i>An algebra of variant loci</i></a>
                                <li><a class="hiddenlink" href="posters/eccb-2014.pdf">European Conference on Computational Biology 2014. <i>Massively parallel read mapping on GPUs with the q-group index and PEANUT.</i></a>
                                    <li><a class="hiddenlink" href="posters/gcb-2014.pdf">German Conference on Bioinformatics 2014. <i>An algebra of single nucleotide variants.</i></a>
                                        <li><a class="hiddenlink" href="posters/forschungstag-2014.pdf">University Hospital Essen, Science Day 2014. <i>An algebra of single nucleotide variants.</i></a>
                                            <li><a class="hiddenlink" href="posters/ccss-ude-2013.pptx">Center for Computational Sciences and Simulation, University Duisburg-Essen 2013. <i>DNA-Sequenzanalysen mit GPUs.</i></a>
                                                <li><a class="hiddenlink" href="posters/forschungstag-2013.pdf">University Hospital Essen, Science Day 2013. <i>Massively parallel read mapping with GPUs.</i></a>
                                                    <li><a class="hiddenlink" href="posters/jk-qpcr-ngs-freising-2013.pdf">qPCR-NGS Meeting in Freising 2013. <i>Bioinformatics Workflows with Snakemake.</i> </a></li>
                                                    <li><a class="hiddenlink" href="posters/jk-hypernetworks-2011.pdf">University Hospital Essen, Science Day 2011. <i>Protein hypernetworks.</i></a></li>
                </ul>
            </section>
            <section>
                <h4>Awards and Grants</h4>
                <ul>
                    <li><a href="https://www.cwi.nl/news/2016/veni-grants-l%C3%A9o-ducas-and-johannes-k%C3%B6ster">NWO Veni grant (€ 250,000)</a> for the project "Fully reproducible workflows scaling from workstations to the cloud"</li>
                    <li><a href="http://www.cs.tu-dortmund.de/nps/de/Aktuelles/Newsarchiv/2011/2011_04_11_Uhde_Preis_2011/index.html">Uhde-Award 2011</a> for my diploma thesis "Propagating Interaction Logic towards Predictive Protein Hypernetworks".</li>
                    <li>Honorable Mention at the Doktorandenkolleg Ruhr 2011 for my poster "Protein Hypernetworks".</li>
                    <li>Poster Award of the University Hospital Essen at the Forschunstag 2011 for my poster "Protein Hypernetworks".</li>
                    <li>Travel Award of the 9th International Conference on Pathways, Networks and Systems Medicine 2011 for my talk on protein hypernetworks.</li>
                </ul>
            </section>

            <section>
                <h4>Teaching</h4>
                <table class="table">
                    <tbody>
                        <tr>
                            <th>2013</th>
                            <td>Guest lecture "Detecting SNVs with Next-generation-Sequencing" in the course "Statistik in der Genetik", Faculty of Statistics, TU Dortmund.
                                <tr>
                                    <th>2012</th>
                                    <td>Co-supervised bachelor thesis "Rekonstruktion von Protein-Interaktionsabhängigkeiten mit dem Quine-McCluskey-Algorithmus", Bianca Patro, TU Dortmund.</td>
                                </tr>
                                <tr>
                                    <th>2011</th>
                                    <td>Teaching assistant for "Datenstrukturen Algorithmen und Programmierung" (DAP1), Faculty of Computer Science, TU Dortmund<br/> Co-supervised bachelor thesis "Konstruktion von Protein-Hypernetzwerken durch Text-Mining
                                        in der PubMed Datenbank", Michael Nimbs, TU Dortmund.<br/> Co-supervised diploma thesis "Entwurf einer Datenstruktur für Pangenome", Christiane Küch, TU Dortmund.</td>
                                </tr>
                    </tbody>
                </table>
            </section>
        </section>
        <section id="contact">
            <h3>Contact</h3>
            <table class="table">
                <tbody>
                    <tr>
                        <th>email</th>
                        <td><a href="mailto:johannes.koester@uni-due.de">johannes.koester@uni-due.de</a></td>
                    </tr>
                    <tr>
                        <th>phone</th>
                        <td>+49 (0)201 723 1908</td>
                    </tr>
                    <tr>
                        <th>www</th>
                        <td><a href="https://koesterlab.github.io">https://koesterlab.github.io</a></td>
                    </tr>
                    <tr>
                        <th>keybase</th>
                        <td><a href="https://keybase.io/johanneskoester">johanneskoester</a></td>
                    </tr>
                    <tr>
                        <th>office</th>
                        <td>
                            Room 1.13 University Hospital Essen Virchowstr. 183 45147 Essen
                        </td>
                    </tr>
                    <tr>
                        <th>postal</th>
                        <td>
                            Dr. rer. nat. Johannes Köster<br/> Institute of Human Genetics<br/> University of Duisburg-Essen<br/> Hufelandstr. 55<br/> 45147 Essen<br/> Germany
                        </td>
                    </tr>
                </tbody>
            </table>
        </section>
    </div>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.0.0-rc1/js/bootstrap.min.js"></script>
</body>

</html>
